backend/app/assets/stylesheets/spree/backend/globals/_fonts.scss

```diff
@@ -0,0 +1,16 @@
+@font-face {
+  font-family: 'Inter';
+  font-style:  normal;
+  font-weight: 400;
+  src: url("Inter-Regular.woff2") format("woff2"),
+       url("Inter-Regular.woff") format("woff");
+}
+
+@font-face {
+  font-family: 'Inter';
+  font-style:  normal;
+  font-weight: 600;
```

Comment:
We should use a font weight variable here that is then used wherever we use this font face and weight